SpaceInvadersProject/Assets/Scripts/Controllers/PlayerController.cs

```diff
@@ -6,6 +7,8 @@ namespace SpaceInvaders
     [RequireComponent(typeof(Animator))]
     public class PlayerController : EntityController
     {
+        public event EventHandler Exploding;
```

Comment:
I added this event to abort the Omega Ray as soon as the player explodes, since the `Destroyed` event is raised only after the explosion animation ends.

The commit message incorrectly called it `Destroying`.